### ISSUE TITLE
Train Resnet50 on Serengeti Data

### ISSUE THREAD

**Issue by Zoyela** [Zoyela/DeepDIVA#14]
* 

- Learning rate .01
-     Momentum 0.9
-     epochs: look at learing behaviour, it takes about 15 minutes, maybe 50 epochs
-     checkpoints default - best and latest
-     use the model pretrained on imagenet
-     provided by deepDIVA
-     look at validation curve - if it still looks like going up, then we should train more
-     this is just pretraining
-     Train with distinction between animals
-     nvidia-smi, use the others that are not in use
-     run inside tmux or scream
-- tmux

 new -s temp
